src/olympia/reviewers/tests/test_views.py

```diff
@@ -6096,6 +6096,25 @@ def test_redirect_after_review_unlisted(self):
 
         self.assertRedirects(response, self.url)
 
+    def test_review_moderator_addon_rating_read_only(self):
+        user = user_factory()
+        Rating.objects.create(
+            body='Lorem ipsum dolor',
+            rating=3,
+            ip_address='10.5.6.7',
+            addon=self.addon,
+            user=user,
+        )
+
+        self.grant_permission(self.reviewer, 'Rating:Modarate')
```

Comment:
it's `'Ratings:Moderate'`

Reply:
The fact that tests passed despite this typo is suspicious...

Reply:
I think we should test with and without a mozilla.com address and specifically check whether the element is a link or not.

Reply:
I was suspicious too, but in this case both the permission and the email address need to be correct so it was functioning correctly (though doesn't make a great test)

Reply:
we do test with a mozilla email address in `test_review_moderator_addon_rating_present`.  It might be safer though, to have a single function that sets everything up, then renders it once with a @mozilla address and once with a non mozilla email.

Reply:
So it's better to merge ```test_review_moderator_addon_rating_present``` and the one I created on only one test, and then verify specifically the link instead of checking for ```assert doc('.addon-rating strong')``` as I did when rendering with the non mozilla email?

Reply:
Add to the existing `test_review_moderator_addon_rating_present` function - after the existing code change the email address of the user to one that doesn't end in `@mozilla.com`,  continue with `response = self.client.get(self.url)` and so on like you did in `test_review_moderator_addon_rating_read_only`.  That way the rest of the setup is identical, and only the email is different.